client/src/routes/routes.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 
-const Login = React.lazy(() => import('../pages/Login')) 
+const Login = React.lazy(() => import('../pages/login')) 
 const RegisterAccount = React.lazy(() => import('../pages/RegisterAccount')) 
 const RegisterProfile = React.lazy(() => import('../pages/RegisterProfile')) 
 
 const Collection = React.lazy(() => import('../pages/Collection'))
+const DetailProduct = React.lazy(() => import('../pages/DetailProduct'))
```

Comment:
Di Repo kita sebelumnya ini const CollectionDetail = React.lazy(() => import('../pages/CollectionDetail')), jadi ttp pake ini yah pang, jangan diubah lagi, to solve dirimu ubah nama file DetailProduct.jsx jadi CollectionDetail.jsx yah